sdk/helper/certutil/helpers.go

```diff
@@ -2150,3 +2156,101 @@ func IsPSS(algorithm x509.SignatureAlgorithm) bool {
 		return false
 	}
 }
+
+// Per RFC 5280 4.2.1.15; the same as distribution point
+type deltaDistributionPoint struct {
+	DistributionPoint distributionPointName `asn1:"optional,tag:0"`
+	Reason            asn1.BitString        `asn1:"optional,tag:1"`
+	CRLIssuer         asn1.RawValue         `asn1:"optional,tag:2"`
+}
+type distributionPointName struct {
+	FullName     []asn1.RawValue  `asn1:"optional,tag:0"`
+	RelativeName pkix.RDNSequence `asn1:"optional,tag:1"`
+}
+
+func CreateDeltaCRLExtension(deltaUrls []string) (pkix.Extension, error) {
```

Comment:
Is there any valid reason to create a DeltaCRL without an url? If not should this method error out if pass a nil or empty slice for deltaUrls instead of creating an extension without a value?